MapboxNavigation/NavigationView.swift

```diff
@@ -163,18 +148,75 @@ open class NavigationView: UIView {
         self.commonInit()
     }
     
+    // MARK: - NavigationView
+    
+    override open func prepareForInterfaceBuilder() {
```

Comment:
Did any of this block change or was it just moved?

Reply:
just moved